src/bloxone_client/api_client.py

```diff
@@ -25,6 +25,12 @@
 RequestSerialized = Tuple[str, str, Dict[str, str], Optional[str], List[str]]
 
 
+header_client = "x-infoblox-client"
+header_sdk = "x-infoblox-sdk"
+sdk_identifier = "python-sdk"
```

Comment:
since they are constants, they should be capitalized.

Reply:
Modified